### ISSUE TITLE
Status reacts 2

### ISSUE THREAD

**Issue by ashfordneil** [UQComputingSociety/uqcsbot#309]
Can we please add an extra status react for when handling a command throws an exception? Right now we can’t tell whether something has failed or if it’s blocking somehow.

**Comment by mitchmcdee** [comment]
Yeah I somewhat started this idea in #304 with the addition of `FailureException` for success reacts, will look to expand on the idea 👍
